etc/errors/anvil.json

```diff
@@ -19,7 +19,215 @@
           "component_name": "AnvilEnvironment",
           "component_code": 1,
           "identifier_encoding": "env",
-          "errors" : []
+          "errors" : [
+            {
+              "name": "InvalidArguments",
+              "code": 1,
+              "message": "Invalid arguments: {details}.",
+              "fields": [
+                {
+                  "name": "details",
+                  "type": "string"
+                },
+                {
+                  "name": "arguments",
+                  "type": "string"
+                }
+              ],
+              "doc": {
+                "summary": "Invalid command line arguments provided.",
+                "description": "There are missing or invalid command line arguments, or an invalid combination of arguments is provided.",
+                "likely_causes": [
+                  {
+                    "cause": "Missing an essential argument or an incorrect value is provided.",
+                    "fixes": [
+                      "Check the short help on the CLI arguments using `anvil-zksync -h`"
+                    ]
+                  }
+                ]
+              }
+            },
+            {
+              "name": "ServerStartupFailed",
+              "code": 2,
+              "message": "Failed to start server at {host_requested}:{port_requested}, and failed to fall back to {host_requested}:{port_fallback_retried}: {details}.",
+              "fields": [
+                {
+                  "name": "host_requested",
+                  "type": "string"
+                },
+                {
+                  "name": "port_requested",
+                  "type": "uint"
+                },
+                {
+                  "name": "port_fallback_retried",
+                  "type": "uint"
+                },
+                {
+                  "name": "details",
+                  "type": "string"
+                }
+              ],
+              "doc": {
+                "summary": "Failed to start the server and bind it to the requested host and port.",
+                "description": [
+                  "Anvil-zksync starts the server and listens to requests on a specified host and port, 0.0.0.0:8011 by default. They are configurable using `--host` and `--port` command line arguments.",
+                  "",
+                  "The host and port used by anvil-zksync are also displayed when you start anvil-zksync:",
+                  "",
+                  "```",
+                  "========================================",
+                  "Listening on 0.0.0.0:8011",
+                  "========================================",
+                  "```",
+                  "",
+                  "This error indicates that listening on the specified host and port There are missing or invalid command line arguments, or an invalid combination of arguments is provided."
```

Comment:
Looks like you didn't finish the first sentence in this line

Reply:
Oops, thank you!